lib/crypto/butil.js

```diff
@@ -35,7 +35,7 @@ module.exports.xorBuffers = function xorBuffers(buffer1, buffer2) {
 module.exports.unbuffer = function unbuffer(object, inplace) {
   var keys = Object.keys(object)
   var copy = inplace ? object : {}
-  for (var i = 0; i < keys.length; i++) {
+  for (var i = 0; i < keys.length; i ++) {
```

Comment:
Oh look, a bikeshed! I personally hate the space in unary operators, pluses and bangs.